src/textual/geometry.py

```diff
@@ -631,18 +628,58 @@ def css(self) -> str:
     def unpack(cls, pad: SpacingDimensions) -> Spacing:
         """Unpack padding specified in CSS style."""
         if isinstance(pad, int):
-            return cls(pad, pad, pad, pad)
+            return Spacing.all(pad)
```

Comment:
I'd rather not use this here. It's a tiny but slower, and I don't think it benefits much from readability. I try to keeps functions at this level as atomic as possible.